raiden-dapp/src/components/AppHeader.vue

```diff
@@ -26,7 +26,9 @@
             <div class="app-header__top__content__network">{{ network }}</div>
           </v-col>
           <v-spacer></v-spacer>
-          <header-identicon />
+          <div @click="navigateToGeneralHome()">
```

Comment:
Didn't this work on just the `<header-identicon @click="..." />`? If not, maybe `<header-identicon @click.native="..." />` does the trick

Reply:
Honestly, I did not event think about that. Will change it.